### PR TITLE
Rename ElasticIP to Address and bump to v1beta1

### PR DESCRIPTION
**Pull request by enderv**
Signed-off-by: chris <cvodak@ea.com>

### Description of your changes
Rename ElasticIP to Address and bump to v1beta1

Fixes #

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
Ran unit tests. Created example using locally run

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: apis/ec2/v1beta1/referencers.go [hunk 1]
```diff
@@ -209,19 +207,19 @@ func (mg *NATGateway) ResolveReferences(ctx context.Context, c client.Reader) er
 	mg.Spec.ForProvider.SubnetID = aws.String(subnetIDResponse.ResolvedValue)
 	mg.Spec.ForProvider.SubnetIDRef = subnetIDResponse.ResolvedReference
 
-	// // Resolve spec.elasticIp
-	AllocationIDRespone, err := r.Resolve(ctx, reference.ResolutionRequest{
+	/*// // Resolve spec.elasticIp
```
Comment (hasheddan) [inline apis/ec2/v1beta1/referencers.go:210]: Is this intentionally commented out?

Comment (enderv) [reply apis/ec2/v1beta1/referencers.go:210]: sigh. had to comment it out to get generate going. am fixing

**Comment by enderv** [comment]
@hasheddan when ya get a chance anything else needed on this one?

**Review by hasheddan — APPROVED** [review]
Thanks @enderv! LGTM 🎉

**Pull request merged**